addons/account/models/account_payment.py

```diff
@@ -2,7 +2,9 @@
 from odoo import models, fields, api, _, Command
 from odoo.exceptions import UserError, ValidationError
 from odoo.tools.misc import format_date, formatLang
+from odoo.tools import SQL
 
+from collections import defaultdict
```

Comment:
Don't follow the coding guideline for import :smile: https://www.odoo.com/documentation/17.0/contributing/development/coding_guidelines.html#imports